Source/Core/Common/GL/GLInterface/AGL.mm

```diff
@@ -5,31 +5,54 @@
 #include "Common/GL/GLInterface/AGL.h"
 #include "Common/Logging/Log.h"
 
-void cInterfaceAGL::Swap()
+static bool DimensionsWereUpdated(NSView* view, u32* width, u32* height)
```

Comment:
This should likely have a better name. This currently reads as something that solely checks values but doesn't modify them at call sites.